gradle.properties

```
@@ -19,7 +19,7 @@ android.useAndroidX=true
 android.enableJetifier=false
 
 GROUP=io.github.david-allison-1
-VERSION_NAME=0.1.11
+VERSION_NAME=0.1.13-anki2.1.53
```

Just curious if there is a reason it went to 0.1.13-nnn vs 0.1.12-nnn ? speculative suggestion based on the thought

```suggestion
VERSION_NAME=0.1.12-anki2.1.53
```

---

I bumped it once at the start, and again after I changed the backend API, just to make it clearer that the API had changed. Feel free to adjust as desired :-)

---

okay, doesn't really matter in the grand scheme, version numbers are free and every commit has a cost, so we go with 13, done!